kalite/distributed/templates/distributed/about.html

```diff
@@ -0,0 +1,25 @@
+{% extends "distributed/base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+
+{% block title %}{% trans "About" %}{% endblock title %}
+{% block headjs %}
+    {{ block.super }}
+    <script src="{% static 'js/distributed/bundles/bundle_about.js' %}"></script>
+    <script>
+        $(function() {
+            var about_view = new window.AboutView({
+                el: $("#about-diagnostics-container")
+            });
+        });
+    </script>
+{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <h2>Diagnostics</h2>
+        <div id="about-diagnostics-container">
+            <h4>Calculating diagnostics...</h4>
```

Comment:
Also, perhaps the message is better as:

```html
<p><em>{% trans "Calculating diagnostics..." %}</em></p>
<p>{% trans "This can take several minutes." %}</p>
```